client/templates/topic/topicInfoItemEdit.js

```diff
@@ -237,7 +237,8 @@ Template.topicInfoItemEdit.events({
         cancelButton.prop('disabled',false);
 
         let editItem = getEditInfoItem();
-        tmpl.find('#id_item_subject').value = (editItem) ? editItem._infoItemDoc.subject : '';
+        let infoItem = tmpl.find("#id_item_subject");
+        infoItem.value = (editItem) ? editItem._infoItemDoc.subject : "Info";
```

Comment:
Check the acceptance criteria of your user story. We want this behaviour only for InfoItems. Unfortunately this "Info" text also shows up when you create a new ActionItem.